.github/workflows/stale.yml

```diff
@@ -0,0 +1,44 @@
+# GitHub Action reference: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions
+#
+# Automated handling of stale issues and pull requests.
+#
+# NOTE: When changing "days-before-*" values, please also update the "stale-issue-message" and/or
+# "stale-pr-message" values.
+
+name: stale
+
+on:
+  schedule:
+    #        ┌───────────── minute (0 - 59)
+    #        │ ┌───────────── hour (0 - 23)
+    #        │ │ ┌───────────── day of the month (1 - 31)
+    #        │ │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+    #        │ │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+    #        * * * * *
+    - cron: '8 3 * * *'
```

Comment:
that's a very specific time for such a cron job 😛

Reply:
lol just applying some random jitter